Algorithms/Shors/shors.n512.scaffold

```diff
@@ -1015,16 +1015,16 @@ module RotBy(qbit q, int k, int neg)
 module CRinv(qbit ctrl, qbit q, int k) //Controlled RotBy
 {
   RotBy(q,k-1, -1); //SP: negative angle
-  CNOT(q, ctrl);  // ajavadia: changed order of ctrl and target
+  CNOT(ctrl, q);  // ajavadia: changed order of ctrl and target
```

Comment:
Could you remove this comment? I'm guessing there was some back and forth on the control target ordering in the past too lol.